docs/features/series.md

```diff
@@ -0,0 +1,205 @@
+# Series
+Series enables customers to bundle episodic content such as TV shows and learning courses or non-episodic content like sports leagues events.  By organizing content into a series, viewers are guided through the content. Series have a predefined sequence of episodes and can be split in seasons.
+
+<img title="" src="img/series.jpg" alt="Series" width="580">
+
+Series are tagged with `Series` in [shelves and libraries](shelves-and-libraries.md):
+
+<img title="" src="img/series-in-library.jpg" alt="Series in library" width="581">
+
+Series are defined through 'series playlist'. This is handled in the first piece of this article.
+
+In the near future JW player will native series construct. This is handled in the second part of this article.
+
+## Series through playlist
+Series are not a native construct in the JW Dashboard at this moment. So customers create a 'series playlist' and set the sequences and episodes using custom parameters.
+
+### Creating series playlists in the dashboard
+The [JW manual](https://support.jwplayer.com/articles/build-an-ott-apps-series-playlist) describes the following process to create a serie playlist: 
```

Comment:
Can we just link to the external resource and not add too much specific detail here?  My worry is that this will get out of date at some point and no one will think to come here to fix it.

Reply:
agreed